packages/core/android/src/test/kotlin/com/datadog/reactnative/DdRumTest.kt

```diff
@@ -63,8 +64,8 @@ internal class DdRumTest {
 
     lateinit var fakeContext: Map<String, Any?>
 
-    @LongForgery(1000000000000L, 2000000000000L)
-    var fakeTimestamp: Long = 0L
+    @DoubleForgery(1000000000000.0, 2000000000000.0)
+    var fakeTimestamp: Double = 0.0
```

Comment:
we could do it the other way: keep it as `Long` and then just call `toDouble` where needed, seems less changes in this case.